webapp-django/crashstats/signature/jinja2/signature/signature_aggregation.html

```diff
@@ -3,7 +3,7 @@
     <thead>
       <tr>
         <th scope="col">Rank</th>
-        <th scope="col">{{ aggregation | replace('_', ' ') | capitalize }}</th>
+        <th data-field-name="{{ aggregation }}" scope="col">{{ aggregation | replace('_', ' ') | capitalize }}</th>
```

Comment:
This fixes sorting fields that jquery-tablesorter has difficulties sussing the type and sort method for. I generalized the `fieldOverrides` nonsense, but need to capture the field name in a value that I can carry through.

We use the aggregation because it's the super search field name in snake-case.